build-scripts/ose_images/ose_images.sh

```diff
@@ -1811,6 +1835,12 @@ case "$action" in
     echo "Fail Pushes: ${BUILD_FAIL}"
     cat ${workingdir}/logs/buildfailed | cut -d':' -f3-4
   ;;
+  scan_images )
+    popd > /dev/null
+    sudo atomic scan --scanner openscap \
+      $(< "${workingdir}/images_to_scan.txt") \
+      > scan.txt
```

Comment:
What is the best way to get this to the machine that's going to execute the job?

`atomic scan` requires, obviously, the `atomic` command, and also the
`openscap` scanner image.  The steps I used to install on my fedora machine
were:

```
# dnf install atomic
# atomic install registry.access.redhat.com/rhel7/openscap

Reply:
The [docs][docs] say

> The OVAL definitions used by the CVE scan type are bundled in the container
> image during the build process, and as such are not always up-to-date. 

so we should keep the image up to date.  We could do a pull before each scan.

[docs]: https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Security_Guide/sect-Using_OpenSCAP_with_Atomic.html

Reply:
Also, can the jobs execute as the root user on this jenkins instance?  `atomic
scan` does a lot of black magic and has to execute as root.

Reply:
@bbguimaraes You should be able to use sudo in this environment to update the image.

Reply:
Command to update the image added.

Reply:
Thanks @jupierce I have installed that on the new build vm.